CONTRIBUTING.md

```diff
@@ -0,0 +1,156 @@
+
+As a content owner, you'll need the following tools in order to successfuly contribute to the project: 
+
+* The [git](https://git-scm.com/) open source distributed version control system.
+* A [github](http://github.com) account and access to the [Intersys Consulting](https://github.com/intersysconsulting) organization on the platform.
+* At least Java 8 and the Scala Build Tool [SBT](https://www.scala-sbt.org/1.x/docs/Setup.html).
+* The [Jekyll](https://jekyllrb.com/) site generator. 
+* [Pandoc](https://pandoc.org/), the file converter tool. 
+
+A setup script on a **debian-based OS** should look similar to:
+```bash
+#!/bin/bash
+
+# Install git
+sudo apt install git
+
+# Install java 8
+sudo apt install openjdk-8-jdk
+
+# Install SBT
+echo "deb https://dl.bintray.com/sbt/debian /" | sudo tee -a /etc/apt/sources.list.d/sbt.list
+sudo apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv 2EE0EA64E40A89B84B2DF73499E82A75642AC823
+sudo apt update
+sudo apt install sbt
+sbt about
+
+# Install Ruby and Jekyll
+sudo apt install ruby ruby-dev build-essential
+export GEM_HOME=$HOME/gems
+export PATH=$HOME/gems/bin:$PATH
+gem install jekyll bundler
+
+# Install pandoc
+sudo apt install pandoc
+```
+
+## Github Repository
+
+The nanodegree's content and learning material will be hosted in [this github repository](https://github.com/IntersysConsulting/scala-essentials-nanodegree). 
+
+Consider the following relevant dirs and docs:
+* `publish.sh` is a bash script used to publish the content microsite.
+* `src/main/tut/` contains the markdown material of the microsite
+    * `index.md` is the landing page of the microsite.
+    * `authors.md` 
+    * `docs/` contains the markdown resources of the modules.
+        * `moduleX.md` is the content of the module X.
+* `slides/` contains the makdown resources of the slides.
+    * `moduleX.md` is the content of the module X. 
+**Repository structure**:
+
+```text
+|- README.md
+|- LICENSE.md
+|- .gitignore
+|- publish.sh
+|- build.sbt
+|- project/...
+|- slides/
+    |- module1.md
+    |- module2.md
+    |- ...
+|- src/
+    |- main/
+        |- scala/
+        |    |- com.intersysconsulting.nanodegrees.scalaessentials
+        |        |- examples/
+        |        |    |- module1/...
+        |        |    |- module2/...
+        |        |    |- ...
+        |        |- challenge/...
+        |- resources/
+        |    |- microsite/
+        |        |- data/
+        |        |    |- menu.yml
+        |        |- slides/...
+        |        |- js/...
+        |        |- img/...
+        |- tut/
+            |- docs/
+            |    |- index.md
+            |    |- module1.md
+            |    |- module2.md
+            |    |- module3.md
+            |    |- ...
+            |- index.md
+            |- docs.md
+            |- slides.md
+            
+```
+## Development
+
+The development of each module should occur on a separate and individual branch that derives from `develop` 

```

Comment:
The branch naming convention remains me to git-flow, if there a plan to follow such a versioning style let's be explicit bout it.

Reply:
Makes sense!